### PR TITLE
Add option to run all the scan at once

### LINKED ISSUE #46: Implement 'all' scan feature
User now need to run the scans individually, but we can implement a new feature which will allow the user to run the scans in one go:

A. `python3 vault.py -u 'http://url.example.com' -all` will perform
1. SSL Scan
2. XSS RFI LFI SQLi
3. Info gathering, comment finding, testing HTTP methods, Clickjacking
4. Fuzzing
5. Crawling

B. `python3 vault.py -ip '10.0.2.15' -all` will perform

1. IP Scanner
2. Port Scanner

### PR DESCRIPTION
**Pull request by mzfr**
#### Changes this pull request makes :

Fixes #46 
Also fixed some small pep8 issues

#### Checklist :

  - [ ] I have added the relevant documentation.
  - [x] My branch is up-to-date with the upstream master branch.
  - [x] I have fully tested the changes locally.

#### Development environment

    OS: Manjaro
    Python Version: 3.5

### CODE REVIEW

File: src/vault.py [hunk 1]
```diff
@@ -438,6 +438,25 @@ def ddos(args):
 
     args = parser.parse_args()
 
+    if args.all:
+        if args.url:
+            url = check_url(args.url)
```
Comment (abhisharma404) [inline src/vault.py]: What's the need to store URL in another variable? Instead `args.url = check_url(args.url)`

File: src/vault.py [hunk 2]
```diff
@@ -438,6 +438,25 @@ def ddos(args):
...
+            ssl(args)
+            info(args)
+            fuzz(args)
+            comment(args)
+            xss(args)
+            lfi(args)
+
+        if args.ip:
+            ip = check_ip(args.ip)
```
Comment (abhisharma404) [inline src/vault.py]: Same here.
Instead a better solution would be to add `args.all` after `args.url` and `args.ip` have been validated. (After Line 445 (master branch))

File: src/vault.py (hunk 1 above)
Comment (abhisharma404) [reply src/vault.py]: @mzfr Actually we won't be using `url` variable anywhere, what gets passed to other modules is `args.url` and `args.ip`. So it was not needed! :)

**Comment by mzfr** [comment]
@abhisharma404 all the requested changes have been made. Please review.

**Pull request merged**